### PR TITLE
Fix issue #9

### PR DESCRIPTION
**Pull request by monicachew**
Add blushlist.js so we can keep the data separate. Change matchBlushlist to getCategoryForBlushlist and make it use eTLD instead of substring matching.

### CODE REVIEW

**Comment by mozkeeler** [comment]
Should this go in data/ somehow? (that might be more trouble than it's worth, though)
Also, do you want to modify bpCategorizer.js to use this new file?

**Comment by monicachew** [comment]
Hi David,

My understanding is that files in data are loaded once per page via pagemod. This should just be loaded when the extension is loaded. Also, I got the pull request wrong, there are mods in bpCategorizer, dunno why they didn't get pulled in. I'll fix it.

Monica

**Comment by monicachew** [comment]
Oh, I forgot to push after I committed to my branch. I think the diff is correct now.

**Comment by monicachew** [comment]
Now I have to figure out how to merge branches...

File: lib/bpCategorizer.js [hunk 1]
```diff
@@ -4,33 +4,25 @@
...
-  ["xxx",'porn'],
-  ["okcupid",'dating'],
-  ['poker','gambling']
-];
+var {Cc, Ci} = require("chrome");
+var eTLDService = Cc["@mozilla.org/network/effective-tld-service;1"].
+                  getService(Ci.nsIEffectiveTLDService);
 
-/** Return the first blushlist tuple matching the url.
+const blushlist = require("./blushlist");
```
Comment (mozkeeler) [inline lib/bpCategorizer.js]: I think it's just 'require("blushlist")'

Comment (monicachew) [inline lib/bpCategorizer.js]: Fixed, please have another look.

**Comment by mozkeeler** [comment]
Great - thanks!

**Pull request merged**